config/ConfigExample/RemoteConfigViewController.swift

```diff
@@ -70,6 +70,20 @@ class RemoteConfigViewController: UIViewController {
     let settings = RemoteConfigSettings()
     settings.minimumFetchInterval = 0
     remoteConfig.configSettings = settings
+
+    // [START add_config_update_listener]
+    remoteConfig.addOnConfigUpdateListener { configUpdate, error in
+      guard error == nil else { return self.displayError(error) }
+      self.remoteConfig.activate() { changed, error in
+        guard error == nil else { return self.displayError(error) }
+
+        print("Updated keys: \(configUpdate!.updatedKeys)")
```

Comment:
Let's log the updated keys before calling `activate()` (between current lines 76 and 77) since they're available and the intention is to use them to selectively activate.